modules/administration_manual/pages/configuration/files/external_storage_configuration_gui.adoc

```diff
@@ -130,13 +130,13 @@ the respective credentials needed for configuring the connection.
 
 We recommend xref:configuration/server/background_jobs_configuration.adoc[configuring the background job Webcron or Cron] to enable ownCloud to automatically detect files added to your external storages.
 
-TIP: You cannot scan/detect changed files on external storage mounts when you select the 
-*Log-in credentials, save in session* authentication mechanism. However, there is a workaround, 
-and that is to use Ajax cron mode. 
+TIP: You cannot scan/detect changed files on external storage mounts when you select the
```

Comment:
`TIP:` --> `[TIP:]`
(to render properly)

Reply:
I don't wanted to do structural changes, that have just been touched because my editor drops trailing whitespaces on save...

Reply:
`TIP:` would work just fine. `[TIP:]` is not valid AsciiDoc.